hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestRecursiveAclWithFSO.java

```diff
@@ -199,8 +200,14 @@ public void testKeyDeleteAndRenameWithoutPermission() throws Exception {
       // Remove acl from directory c2, delete/rename a/b1 should throw
       // permission denied since c2 is a subdirectory
 
-      UserGroupInformation.setLoginUser(user1);
-      removeAclsFromKey(objectStore, ozoneBucket, "a/b1/c2");
+      user1.doAs((PrivilegedExceptionAction<Void>) () -> {
+        try (OzoneClient c = cluster.newClient()) {
+          ObjectStore o = c.getObjectStore();
+          OzoneBucket b = o.getVolume("volume1").getBucket("bucket1");
+          removeAclsFromKey(o, b, "a/b1/c2");
+        }
+        return null;
+      });
```

Comment:
Please try to extract parts of `testKeyDeleteAndRenameWithoutPermission()`, checkstyle says it's too long:

```
hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestRecursiveAclWithFSO.java
 84: Method testKeyDeleteAndRenameWithoutPermission length is 159 lines (max allowed is 150).
```

https://github.com/whbing/ozone/actions/runs/5032997814/jobs/9026948835#step:6:520